packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1426,43 +1360,67 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       return;
     }
 
+    if (isArrowKey) {
```

Comment:
Shouldn't the ArrowDown case open the suggestions list the same way clicking in the combo-box input field does for consistency of behavior with the FluentUI Combo-Box control?

Reply:
Per the guidance here: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/ (if I am reading it correctly) the down arrow would move focus to the pop up if it is already displayed. optionally if the pop is not displayed, there is an interaction pattern of using alt + down arrow to open the pop up (without changing focus.). However, if Fluent is following a different pattern, would probably make sense to implement as they do. Although, not sure it is necessary for this PR assuming these changes will come with the version of the package that uses Fluent controls.

Reply:
This one is pretty easy to fix, and I think it makes sense to follow what Fluent is doing, so I've got the changes for that working locally.

Reply:
Thanks @patmoo , 

My read of that article is that the behavior when the list is closed is quite open for choice:
>The popup is hidden by default, i.e., the default state is collapsed. The conditions that trigger expansion -- display of the popup --are specific to each implementation. Some possible conditions that trigger expansion include:
> - It is displayed when the Down Arrow key is pressed or the Open button is activated. Optionally, if the combobox is editable and contains a string that does not match an allowed value, expansion does not occur.
> - It is displayed when the combobox receives focus even if the combobox is editable and empty.
> - If the combobox is editable, the popup is displayed only if a certain number of characters are typed in the combobox and those characters match some portion of one of the suggested values.
keypreopening of the suggestions is that DownArrow or Enter should open the suggestions list if it is not open 

I read the part on moving focus as being applicable when the list of suggestions is already open. 

So `DownArrow` when the list is closed opens the suggestions but does not select any item in the list, whereas if the list is open the focus is moved.

For reference the Fluent Combo box can be found in a sandbox here: https://fluent-components.azurewebsites.net/?path=/docs/components-combobox--combobox It is necessary to change the autocomplete setting to inline to get the behavior that matches what we're doing

Reply:
Okay, I believe we're handling this correctly now.  The `DownArrow` opens the suggestions but does not select unless the list is open, then it just moves the focus.

I removed a case that I believe is unreachable to do that.